docs/readmes/faq/magma_faq.md

```diff
@@ -84,6 +100,10 @@ This section lists some of the commonly asked questions related to Magma operati
   - Click on **Try it out** button on right hand side.
   - Put in the required inputs and click **Execute**.
 
+### How to verify which services are running in Orc8r-controller?
+  - **Get the controller pods:** `kubectl -n magma get pods`.
+  - **Execute the command in the pod:** `kubectl exec -n magma <CONTROLLER PODNAME> supervisorctl status`.
+
```

Comment:
i may be using old commands, what is that `magma` keyword on kubectl?

Reply:
Will modify it to "kubectl exec -n <NAME SPACE> orc8r-controller-6b85fbdf77-76mmj -- supervisorctl status". This we checked, as per latest format.

Reply:
Will modify it to "kubectl exec -n <NAME SPACE> orc8r-controller-6b85fbdf77-76mmj -- supervisorctl status. This we checked, is as per latest format.